### PR TITLE
Update acm

### PR DESCRIPTION
**Pull request by kodster28**
Helps make info more clear + allows us to deprecate an older article in the KB.

Not the final changes, but getting the article in a better state.

Comparison --> [Old page](https://developers.cloudflare.com/ssl/advanced-certificate-manager)
**New page**
image

### CODE REVIEW

File: products/ssl/src/content/advanced-certificate-manager/index.md [hunk 1]
```diff
@@ -14,232 +29,90 @@ Advanced Certificate Manager is a flexible and customizable way to issue and man
...
\ No newline at end of file
+## Create a certificate in the dashboard
+
+To create a new advanced certificate in the dashboard:
+1. Log into your Cloudflare account and select a domain.
+1. Select **SSL/TLS** > **Edge Certificates**.
+1. Select **Order Advanced Certificate**.
+1. If Cloudflare does not have your billing information, you will need to enter that information.
+1. Enter the following information:
+    - Certificate Authority
```
Comment (crwaters16) [inline products/ssl/src/content/advanced-certificate-manager/index.md:114]: A little capitalization inconsistency in this list - should all these be capped or lowercase? (For example, style guide shows we capitalize Custom Hostname, does that apply for Certificate Hostname as well?)

File: products/ssl/src/content/certificate-transparency-monitoring/index.md [hunk 2]
```diff
@@ -4,58 +4,70 @@ order: 2
...
+Every website must have a certificate to be trusted by major browsers. A certificate is a proof of identity — it says that you are who you say you are. These certificates help browsers like Google Chrome "know" that a connection is secure before presenting content. Certificates are recorded in public **CT logs**, such as Google’s Argon log and Cloudflare’s Nimbus log.
 
-If you enable CT Monitoring, we’ll send you an email whenever your domain is recognized in a CT log. Usually, the certificate we find is legitimate and no further action is required. We send emails so you can double-check for yourself. In rare cases, you may believe a certificate is illegitimate. This is when action is suggested (see below).
+If you enable Certificate Transparency (CT) Monitoring, Cloudflare will send you an email whenever your domain is recognized in a CT log. Usually, these certificates are legitimate and do not require further action. We send emails so you can double check for yourself.
```
Comment (crwaters16) [inline products/ssl/src/content/certificate-transparency-monitoring/index.md]: double-check (with hyphen)

File: products/ssl/src/content/certificate-transparency-monitoring/index.md [hunk 3]
```diff
@@ -4,58 +4,70 @@ order: 2
...
+Alerts are turned off by default. If you want to receive alerts, go to [SSL/TLS](https://dash.cloudflare.com/?to=/:account/:zone/ssl-tls/edge-certificates#ct-alerting-card) in the Cloudflare dashboard.
+
+Features vary by plan:
+- **Free** and **Pro** customers will see a toggle to turn **On** or **Off**. Emails will go to all Cloudflare account members.
+- **Business** and **Enterprise** customers will see an option to **add emails**. These addresses do not have to be tied to Cloudflare accounts. If you want to send emails to more than 10 people, consider setting up an email alias (ct-alerts@yourcompany.com).
+
+To stop receiving alerts, switch the toggle to **Off** or remove your emails from the feature card.
```
Comment (crwaters16) [inline products/ssl/src/content/certificate-transparency-monitoring/index.md:29]: More of a long-term FYI, but once we have alert content in devdocs, we should remove this section (unless there's something specific to ACM about enabling alerts)

**Review by crwaters16 — APPROVED** [review]

File: products/ssl/src/content/certificate-transparency-monitoring/index.md (hunk 3 above)
Comment (kodster28) [reply products/ssl/src/content/certificate-transparency-monitoring/index.md:29]: I think that's a higher-level question. The whole point of this feature is to set up separate alerts. I think it pre-dates the alert notification service.

I'll bring up to the PM to ask though, that's a good callout.

File: products/ssl/src/content/advanced-certificate-manager/index.md (hunk 1 above)
Comment (kodster28) [reply products/ssl/src/content/advanced-certificate-manager/index.md:114]: Ah, so they're actually inconsistent in the UI. I'm just mirroring what they do there.

**Pull request merged**